lib/addons/p5.dom.js

```diff
@@ -1070,8 +1072,8 @@
    * var capture;
    *
    * function setup() {
-   *   createCanvas(480, 120);
-   *   capture = createCapture(VIDEO);
+   *   createCanvas(480, 480);
+   *   capture = createCapture(VIDEO).hide();
```

Comment:
can you break this into two lines so it's super clear?

Reply:
Sure thing.